tests/xua_sim_tests/test_mixer_routing_output/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(test_mixer_routing_output)
+
+set(APP_HW_TARGET test_xs3_600.xn)
+
+set(XMOS_SANDBOX_DIR ${CMAKE_CURRENT_LIST_DIR}/../../../..)
+
+set(APP_INCLUDES src
+                 ${CMAKE_CURRENT_LIST_DIR}/../shared)
+
+set(APP_COMPILER_FLAGS -O3)
+
+include(${CMAKE_CURRENT_LIST_DIR}/../../../examples/deps.cmake)
```

Comment:
Doesn't including it twice just overwrite the deps var?

Reply:
deps.cmake sets the variable and then test_deps.cmake appends to it.